docs/azure/vscodeforweb.md

```diff
@@ -0,0 +1,102 @@
+---
+Order: 5
+Area: azure
+TOCTitle: VS Code for the Web
```

Comment:
Suggest changing the title and also the MetaDescription to something Azure-specific. From an SEO perspective it's competing with the general VS Code for the Web article.